sirepo/package_data/static/js/radia.js

```diff
@@ -3638,7 +3638,7 @@ SIREPO.app.factory('radiaVtkUtils', function(utilities) {
             const tArr = [];
             const tOffset = points.length / 3;
             typeInfo[type].offset = tOffset;
-            points.push(...t.vertices);
+            SIREPO.UTILS.largePush(points, t.vertices);
```

Comment:
> > You might want to look into using concat instead of push? I think that would eliminate your stack problem. I don't know why you would need to push vs concat.
> 
> I came across something similar with `Math.min` so my mind went that direction. I'll try concat (or direct reference) and if it suffices I'll ditch all this...

[1d8dda0](https://github.com/radiasoft/sirepo/pull/5191/commits/1d8dda062e650151cb9d8c6963f1c3665c10b10c)
Concat does the job so I ditched `largePush()`. BTW, I like `push` because IMHO `a.push(...b)` is cleaner than `a = a.conat(b)`.

Reply:
Can you explain why a.push(...b) is cleaner?

Reply:
I general I prefer modifying in place - "here's a thing, now change it". - vs "here's a thing, now make a new thing with the changes, now assign it to the original thing".  c.f. `dict1.update(dict2)`, vs the hypothetical `dict1 = dict1.add(dict2)`. Maybe "cleaner" isn't the right word...

Reply:
In https://github.com/radiasoft/sirepo/issues/5190#issuecomment-1329498599 I note that `...` is recursive and browser-dependent, so regardless of preference it's a bad fit for expanding arrays.